meson_options.txt

```diff
@@ -19,6 +19,4 @@ option('tools', type: 'boolean', value: true)
 
 option('build_subprojects', type : 'boolean', value : true, yield : true)
 
-option('enable-ysnp', type: 'boolean', value: true)
```

Comment:
Right, this seemed like a great idea a long time ago, however, it has not really been exercised, and gone stale.
When removing the meson-option, then also remove it in the code-base:
```
include/xnvme.h
include/xnvme_ysnp.h
```